src/components/MediaSettings/MediaSettings.vue

```diff
@@ -421,16 +422,31 @@ export default {
 			}
 		},
 
-		handleUpdateBackground(background) {
+		closeModalAndConfirmBackground() {
+			if (this.updatedBackground) {
+				this.handleBackgroundUpdate()
+			}
+			this.closeModal()
+		},
+
+		handleBackgroundUpdate() {
```

Comment:
```suggestion
		handleUpdateBackground(background) {
```

I'd pass a value, to make this method more independent